build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -17,6 +17,27 @@ data:
       }
       Write-Host "OVSExt driver has been installed"
     }
+
+    # Check if the VC redistributable is already installed.
+    $OVSRedistDir="$mountPath\openvswitch\redist"
+    if (Test-Path $OVSRedistDir) {
+      $dllFound = $false
+      $paths = $env:PATH -split ';'
+      foreach ($path in $paths) {
+        $dllFiles = Get-ChildItem -Path $path -Filter "vcruntime*.dll" -File -ErrorAction SilentlyContinue
+        if ($dllFiles.Count -gt 0) {
+          $dllFound = $true
+          break
+        }
+      }
+
+      # vcruntime dlls are not installed on the host, then install the binaries.
```

Comment:
We have to install on the Windows host, see this https://github.com/antrea-io/antrea/pull/6325#discussion_r1615411273

Reply:
There is a file `vc_redist_x64.exe` in path "openvswitch\redist" in side the container image, OVS finally requires `vcruntimexxx.dll` on the environment paths. The "vcruntimexxx.dll" is installed via "vc_redist_x64.exe". 
So the logic here is,
1. Check if path "openvswitch/redist" exists in the container, if yes, it means may need to install binary
2. Check if the exe file is already installed on the Windows host by searching vcruntimexxx.dll in the environment paths, if no, install it. This is to avoid duplicated installation after Pod restart/deletion-creation.

Reply:
I read your other comment below and I now understand why we are doing it this way.
You don't see any issue with installing these files on the host directly?
Is there no way to install these files in a custom directory to avoid mutating the host, and then update the PATH accordingly?

On a related note, if the Node already has an older version of the dll (so we skip installation), can this create any compatibility issues?

Reply:
>  if the Node already has an older version of the dll (so we skip installation), can this create any compatibility issues?
I also have this concern about the version conflict. Windows supports installing vc_redist with multiple versions, and all installed software can be queried from registry. So I plan to use another change to check the installed vc_redist versions on the Windows host, if any newer version already exists, I can skip the installation. If this works, we don't need to check the dll existence in environment path, and we can install the vc redist to the default system path. What do you think @antoninbas ?

Reply:
Sounds good.
After looking into it, it seems there is indeed no way to install vc_redist to a custom directory. So it looks like proper version checks as you are suggesting are the way to go. Thanks.